### DISCUSSION TITLE
Who's using Prisma in combination with Supabase Auth? Can you help me to figure out how to set it up?

### DISCUSSION THREAD

**Discussion by lumenwrites** [prisma/prisma#13435] [Q&A]
I want all my models to be defined in my Prisma schema, but I want my authentication to be handled using Supabase. I know how to set up Supabase Auth, and I know how to connect Prisma to Supabase database, but I don't know how to make them work together.

As far as I understand, I should create my own Profiles table (managed by Prisma), and use Supabase trigger function to automatically create a record every time the user signs up and is added to the Users table (managed automatically by Supabase).

But I don't understand how would I be able to create a relationship between these two tables, how would I create a foreign key in my Prisma schema pointing to the Users table, if the Users table is not defined in my schema, but created automatically by Supabase?

Can you help me out?

**Comment by nurul3101** [comment]
Do you get a trigger/event when a user is added by supabase?
If yes, I think you can subscribe to that event and maintain a user table on your database as well and then you can have a relationship between this user and profile table.

**Reply by vladi-stevanovic** [reply]
👋 Hello @lumenwrites  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
